README.md

```diff
@@ -2,26 +2,53 @@
 
 [![Actions Status](https://github.com/ruby/openssl/workflows/CI/badge.svg)](https://github.com/ruby/openssl/actions?workflow=CI)
 
+OpenSSL for Ruby provides access to SSL/TLS and general-purpose cryptography
+based on the OpenSSL library.
 
-OpenSSL provides SSL, TLS and general purpose cryptography. It wraps the
-OpenSSL library.
+OpenSSL for Ruby is sometimes referred to as "openssl" (in all lowercase) or
+"Ruby/OpenSSL" for disambiguation.
```

Comment:
How about the following 1 sentence with bold fonts? I was inspired for the style from the following 2 Wikipedia pages. I may be wrong. But I suppose the quote may not be used in the formal English for this purpose to define the words unlike Japanese brackets "「」".　The quotes may give an impression people that the word is ironic or not truth such as [air quotes](https://www.evilenglish.net/air-quotes/). Anyone, please correct me if I am wrong.

**OpenSSL for Ruby** also called **openssl** in all lowercase or **Ruby/OpenSSL** for disambiguation, provides access to SSL/TLS and general-purpose cryptography
based on the OpenSSL library.

* https://en.wikipedia.org/wiki/Motherboard
  > A motherboard (also called mainboard, main circuit board, MB, mboard, backplane board, base board, system board, mobo; or in Apple computers logic board) is the main [printed circuit board](https://en.wikipedia.org/wiki/Printed_circuit_board) (PCB)...
* https://en.wikipedia.org/wiki/United_Kingdom
  > The United Kingdom of Great Britain and Northern Ireland, commonly known as the United Kingdom (UK) or Britain,[[k]](https://en.wikipedia.org/wiki/United_Kingdom#cite_note-24)[[14]](https://en.wikipedia.org/wiki/United_Kingdom#cite_note-25) is an [island country](https://en.wikipedia.org/wiki/Island_country) in [Northwestern Europe](https://en.wikipedia.org/wiki/Northwestern_Europe) ...

Reply:
I agree using bold font fits better for this purpose. I still want to use two separate sentences because I feel I have too many SSLs in it to my liking.

Reply:
Sure. That's okay.